back-end/lms/views/populate.py

```diff
@@ -467,9 +460,40 @@ def create_achievement_types():
     return types
 
 
+def create_mil_specialty():
+    values = [{
+        '094001': 'Применение наземных подразделений войсковой разведки'
+    }, {
+        '411300':
+            'Эксплуатация и ремонт автоматизированных систем комплексов ' \
+        'баллистических стратегических ракет наземного базирования'
+    }, {
+        '453000':
+            'Организация эксплуатации и ремонта автоматизированных ' \
+        'систем управления и вычислительных комплексов ракетно-' \
+        'космической обороны'
+    }, {
+        '453100':
+            'Математическое и программное обеспечение функционирования ' \
+        'вычислительных комплексов ракетно-космической обороны'
+    }, {
+        '751100': 'Защита информационных технологий'
+    }, {
+        '100182': 'Стрелковые, командир стрелкового отделения'
+    }, {
+        '106646-543': 'Разведывательные, разведчик-оператор СБР, ПСНР'
+    }]
+
+    types = {}
+    for value in values:
+        typ, _ = AchievementType.objects.get_or_create(achievement_type=value)
```

Comment:
Why do you create `AchivementType` object here? I think it should be `MilSpeciality` instead